src/iop/temperature.c

```diff
@@ -1445,6 +1453,17 @@ void gui_init(struct dt_iop_module_t *self)
   g_signal_connect(G_OBJECT(g->scale_g2), "value-changed", G_CALLBACK(rgb_callback), self);
   g_signal_connect(G_OBJECT(g->presets), "value-changed", G_CALLBACK(presets_changed), self);
   g_signal_connect(G_OBJECT(g->finetune), "value-changed", G_CALLBACK(finetune_changed), self);
+
+  gtk_widget_show_all(g->box_enabled);
+  gtk_stack_add_named(GTK_STACK(g->stack), g->box_enabled, "enabled");
+
+  g->label_disabled = gtk_label_new(_("white balance disabled for camera"));
+  gtk_widget_set_halign(g->label_disabled, GTK_ALIGN_START);
+
+  gtk_widget_show_all(g->label_disabled);
```

Comment:
When starting dt i'm now seeing `(darktable:27766): Gtk-WARNING **: Child name 'disabled' not found in GtkStack` warning.

Reply:
Hmm, I'll see what's going on here as well.

Reply:
Yeah the gui_updates was being called before all this was initialized, so I fixed this in the latest patch.